blockmanager.go

```diff
@@ -1837,13 +1838,12 @@ func (b *blockManager) handleNotifyMsg(notification *blockchain.Notification) {
 		//
 		// Also, in the case the RPC server is enabled, stop rebroadcasting any
 		// transactions in the block that were setup to be rebroadcast.
-		txMemPool := b.server.txMemPool
```

Comment:
I'd prefer to just change this to `txMemPool := b.txMemPool` to minimize the diff.